stl/inc/ranges

```diff
@@ -6165,7 +6187,7 @@ namespace ranges {
     class chunk_by_view : public _Cached_position<_Vw, chunk_by_view<_Vw, _Pr>> {

```

Comment:
I don't see changes to `chunk_by_view` specified - is this a defect?

Reply:
I think they were just both added to C++23 around the same time, and the paper was not updated to include `chunk_by_view` - given that `chunk_by_view` requires a predicate, and `indirect_binary_predicate` requires copyability, there are no other changes required (it's already copy_constructible)